buildah/buildah.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: build.knative.dev/v1alpha1
 kind: BuildTemplate
 metadata:
   name: buildah
+  labels:
+    buildtype: dockerfile
```

Comment:
Is this used for anything?

Reply:
Ah, I have a local yaml for building without Dockerfile and was inspecting the plumbing to differentiate on labeled buildtype, like for a selector. I ought to remove it from here for now.